compiler/rustc_mir/src/transform/mod.rs

```diff
@@ -431,7 +431,7 @@ fn mir_drops_elaborated_and_const_checked<'tcx>(
 
         // Do not compute the mir call graph without said call graph actually being used.
         // Keep this in sync with the mir inliner's optimization level.
-        if tcx.sess.opts.debugging_opts.mir_opt_level >= 2 {
+        if tcx.sess.opts.debugging_opts.mir_opt_level >= 1 {
```

Comment:
To get the performance back in check/debug builds, we'll need to make this conditional on `OptLevel::Aggressive`, too.